ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1250,6 +1252,26 @@ def test_ipahealthcheck_verify_perms_for_source_files(self,
         assert data[0]["kw"]["type"] == "mode"
         assert data[0]["kw"]["expected"] == "0600"
 
+    def test_ipahealthcheck_verify_perms_upgrade_log_file(self,
+                                                          modify_permissions):
+        """
+        This testcase runs ipa-server-upgrade command to generate
+        ipa-server-upgrade.log file. Once the file is generated
+        the permissions are modified to  check that correct status
+        message is displayed by healthcheck tool
+        """
+        self.master.run_command(["ipa-server-upgrade"])
```

Comment:
Is the execution of ipa-server-upgrade necessary? If you're just checking permissions you could also just touch the file to create it and it would be a lot faster.

Reply:
@rcritten, yes i modified the test to just touch the file instead of running ipa-server-upgrade command.